tests/console/journalctl.pm

```diff
@@ -147,13 +147,17 @@ sub run {
 
     # Ensure the time is correct, otherwise we might run into issues with the persistent journal
     # See e.g. https://bugzilla.suse.com/show_bug.cgi?id=1182802
+
     if (is_sle("<15")) {
```

Comment:
You can merge the conditions over here. `is_sle("<15") && !is_public_cloud`, I find it personally a bit more readable.

Reply:
Then the else block would screw this completely up.

Reply:
ah yeah, I haven't scrolled that down. Thanks